src/types/index.ts

```diff
@@ -170,6 +170,7 @@ export interface ITokenResolved extends Partial<IToken> {
   isAe?: boolean;
   isPool?: boolean;
   isReceived?: boolean;
+  isCoin?: boolean;
```

Comment:
Maybe the `IAsset` could have a property `assetType` with possible values `'token' | 'coin'`?